bin/runtime/src/lib.rs

```diff
@@ -759,7 +765,6 @@ construct_runtime!(
         UncheckedExtrinsic = UncheckedExtrinsic
     {
         System: frame_system,
-        RandomnessCollectiveFlip: pallet_insecure_randomness_collective_flip,
```

Comment:
this pallet shift will probably make some current tools fail (unless they use new codec approach for dynamic encoding like subxt does); I would consider keeping the original indexing - this can be done by explicitly defining pallet indices within this `construct_runtime` macro (maybe it's sufficient to mark only `Scheduler`, I'm not sure)

I will summon @DamianStraszak here for support or roast

Reply:
I don't know how the explicit indexing works, and if one can have gaps (I would assume so), but since it's quite standard for other chains to use it, I think it would be good to also do it. It requires research ofc... The question is can we add indexing + remove the pallet right away in one update? I think it should be possible, but we should be consistent with the current indexing, which is probably some default one?

Reply:
yes yes, we can do it right away, with gaps etc

Reply:
Right, then it sounds like a good idea :)

Reply:
Done :P